partcad/src/partcad/shape_config_store.py

```
@@ -0,0 +1,22 @@
+# Created: 2025-01-13
+#
+# Licensed under Apache License, Version 2.0.
+#
+
+
+class ShapeConfigStore:
+    vendor: str | None
+    sku: str | None
+    count_per_sku: int
+
+    def __init__(self, final_config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)